### ISSUE TITLE
[ISSUE]: (MAC M1) YFD Information section cuts off at bottom of screen

### ISSUE THREAD

**Issue by Cle4ncuts** [Y-Foundry-Dao/yfd-dapp-core#142]
### Contact Details

clean@yfoundry.io

### What happened?

On the homepage, after opening the actions menu, the menu cuts off at the bottom of the screen once it automatically refreshes. I can reopen the menu to have it fit again, but it cuts off again on the next refresh.

### Web Address where issue occurred

https://dapp-test.yfoundry.io/#

### What did you EXPECT to happen?

I expected that the action menu would fit within my screen space. 

### How would you fix it?

Ensure that the calculations/padding done for the ui menu remains consistent before and after refreshing.

### Deployment

pisco-1 Terra Testnet (Default)

### What browsers are you seeing the problem on?

Brave

### What wallets were you using?

Terra Station

### Screenshots

<img width="1302" alt="Screenshot 2023-02-14 at 3 50 47 PM" src="https://user-images.githubusercontent.com/83686988/218673035-2c2e8e21-ec6a-4fe8-8392-8491788bb269.png">

### Reproduceable?

- [X] I was able to reproduce this issue more than once.

**Comment by crashtestdumb1** [comment]
I dont believe this is an issue anymore. I cannot reproduce at least and am also using a Brave browser. If you open the menu and it gets cut off due to scaling, you are still just able to scroll. I will leave it for the moment to test a little further but think we can close this one.
